ste/init.go

```diff
@@ -432,9 +432,7 @@ func GetJobSummary(jobID common.JobID) common.ListJobSummaryResponse {
 						Dst:            dst,
 						TransferStatus: common.ETransferStatus.Failed(),
 						ErrorCode:      jppt.ErrorCode()}) // TODO: Optimize
-			case common.ETransferStatus.BlobAlreadyExistsFailure(),
-				common.ETransferStatus.FileAlreadyExistsFailure(),
-				common.ETransferStatus.ADLSGen2PathAlreadyExistsFailure():
+			case common.ETransferStatus.SkippedFileAlreadyExists():
```

Comment:
This is a good simplification. Is there anywhere (e.g. `jobs show` results, JSON-format progress messages) where the old enums get exposed outside the app? If so, we should note that is a change in the changelog.

Reply:
It doesn't appear that we've ever used `BlobAlreadyExistsFailure` or `ADLSGen2PathAlreadyExistsFailure`.